src/login.coffee

```diff
@@ -35,6 +35,18 @@ define ['jquery', 'primedia_events', 'jquery.cookie'], ($, events) ->
 
         $("a.logout").click (e) => @_logOut e
 
+    _prefillAccountName: ($div) ->
```

Comment:
Yeah, I know what you mean by the coupling :-\ 

Based on how it's called (from `_bindForms`), I think this is ok for what it needs to do.